packages/cubejs-query-orchestrator/src/orchestrator/PreAggregations.ts

```diff
@@ -1634,7 +1634,7 @@ export class PreAggregations {
           redisPool: this.options.redisPool,
           // Centralized continueWaitTimeout that can be overridden in queueOptions
           continueWaitTimeout: this.options.continueWaitTimeout,
-          ...(await this.options.queueOptions(dataSource)),
+          ...(this.options.queueOptions ? await this.options.queueOptions(dataSource) : {}),
```

Comment:
@cristipp after my changes `this.options.queueOptions` does not require defaults because it is always set and a function. You can find its configuration [here](https://github.com/cube-js/cube.js/blob/master/packages/cubejs-server-core/src/core/OptsHandler.ts#L523).